RecordingApp/app/src/main/AndroidManifest.xml

```diff
@@ -48,11 +48,11 @@
             android:theme="@style/TR"
             />
 
-        <activity android:name=".ProjectManager.ActivityProjectManager"
+        <activity android:name=".ProjectManager.activities.ActivityProjectManager"
```

Comment:
I think I like the convention of including Activity, Fragment, Dialog, etc in the name of the class since they are a more special kind of object. It helps in finding them in tabs. Besides, and this is in general but as a specific example for ease of explanation, the ProjectManager is more of an abstract concept, or section of the app. One class doesn't exactly completely define the ProjectManager, and the ProjectManager is a composition of an Activity, ProjectLists with corresponding Adapters, etc.

Also consider: 
Could rename ChapterCardAdapter to projectmanager.adapter.ChapterCard, but that would cause confusion especially since the ChapterCardAdapter is going to hold ChapterCards which are very different things.